packages/app/src/components/nominationInfo/EditCard.js

```diff
@@ -87,40 +145,65 @@ const EditCard = props => {
               }
             case props.spanishDropdown === label:
               return (
-                <div>
+                <div key={label} className="dropdown-div">
                   <label className={styles.label}>{label}</label>
-                  <select
+                  <Controller
+                    render={({ field: { onChange } }) => {
+                      return (
+                        <Select
+                          options={options}
+                          defaultValue={{
+                            value: props?.formData[label],
+                            label: props?.formData[label],
+                          }}
+                          onChange={(e) => onChange(e.value)}
+                          styles={customStyles}
+                        />
+                      );
+                    }}
                     name={label}
-                    defaultValue={props.formData[label]}
-                    {...props.register(label)}
-                  >
-                    <option value='Yes'>Yes</option>
-                    <option value='No'>No</option>
-                  </select>
+                    control={props.control}
+                  />
                 </div>
               );
               {
                 /* Link opens up provider name's nominations */
               }
             case label === 'Provider Name':
-              return <Provider 
-                        id={props.id}
-                        formData={props.formData}
-                        openWindow={props.openWindow}
-                        label={label}
-                        />
-            // renders diagnosis accross two columns
+              return (
```

Comment:
this and Provider and Other are just formatting/prettier fixes, did not implement any actual changes here FYI